cram_knowrob/cram_semantic_map_utils/package.xml

```diff
@@ -19,13 +19,13 @@
 
   <build_depend>roslisp_utilities</build_depend>
   <build_depend>roslisp</build_depend>
-  <build_depend>cram_json_prolog</build_depend>
+  <!--build_depend>cram_json_prolog</build_depend-->
   <build_depend>cram_utilities</build_depend>
   <build_depend>cl_transforms</build_depend>
 
   <run_depend>roslisp_utilities</run_depend>
   <run_depend>roslisp</run_depend>
-  <run_depend>cram_json_prolog</run_depend>
+  <!--run_depend>cram_json_prolog</run_depend-->
```

Comment:
Getting rid of package.xml dependency on "cram_json_prolog" breaks loading of asdf system "cram-json-prolog". Besides, if the "cram_json_prolog" is just catkin_ignored, "(asdf:load-system :cram-json-prolog)" still works, so it's still being loaded. Please get rid of the catkin_ignore in cram_json_prolog and bring the dependencies on "cram_json_prolog" back in these package.xml-s. As long as the "iai_common_msgs" is there, "cram_json_prolog" doesn't create any problems.